iphone/Maps/Bookmarks/Categories/Permissions/BMCPermissionsHeader.swift

```diff
@@ -28,8 +28,8 @@ final class BMCPermissionsHeader: UIView {
   var delegate: BMCPermissionsHeaderDelegate!
 
   private func updateButton() {
-    UIView.animate(withDuration: kDefaultAnimationDuration) {
-      self.button?.imageView?.transform = self.isCollapsed ? .init(rotationAngle: .pi) : .identity
+    UIView.animate(withDuration: 0.3) {
+      self.button?.imageView?.transform = self.isCollapsed ? .init(rotationAngle: .pi - 0.001) : .identity //fix for rotation direction on expand-collapse
```

Comment:
What's the purpose of ```.pi - 0.001```?

Reply:
Without it button rotates in clockwise direction on both expand and collapse.
I've added it to change rotation direction to counterclockwise on expand.